lib/ts/recipe/passwordless/components/themes/signInUp/linkSent.tsx

```diff
@@ -67,9 +66,6 @@ const PasswordlessLinkSent: React.FC<LinkSentThemeProps> = (props) => {
                 }, 2000);
             } else {
                 setStatus("ERROR");
-                if (response.status === "GENERAL_ERROR") {
-                    props.onError(response.message);
```

Comment:
Shouldn't we make this call in the catch block?

Reply:
it is being made

```ts
try {
    ...

    if (response.status === "OK") {
        setStatus("LINK_RESENT");
        ...
    } else {
        setStatus("ERROR");
    }
} catch (e) {
    setStatus("ERROR");
}
```

Reply:
So the change here was just to remove the GENERAL_ERROR handling, setError was being called originally as well so it probably isnt showing up in git diff

Reply:
I meant the `props.onError(response.message);`